yt/yt/server/node/exec_node/job_prober_service.cpp

```diff
@@ -77,16 +77,17 @@ class TJobProberService
         VERIFY_THREAD_AFFINITY(JobThread);
 
         auto jobId = FromProto<TJobId>(request->job_id());
-        context->SetRequestInfo("JobId: %v", jobId);
+        context->SetRequestInfo("JobId: %v, Limit: %v, Offset: %v", jobId, request->limit(), request->offset());
 
         auto job = Bootstrap_->GetJobController()->FindRecentlyRemovedJob(jobId);
         if (!job) {
             job = Bootstrap_->GetJobController()->GetJobOrThrow(jobId);
         }
 
-        auto stderrData = job->GetStderr().value_or("");
-
-        response->set_stderr_data(stderrData);
+        auto stderrResp = job->GetStderr({.Limit = request->limit(), .Offset = request->offset()}).value_or(NApi::TGetJobStderrResponse{});
```

Comment:
`stderrRsp` – we usually shorten response as rsp

Also let place `.value_or(...)` on the next line for better readability